pages/prelude/index.md

```diff
@@ -5,18 +5,18 @@ datatable: false
 toc: false
 ---
 
-Wizzie Data Platform, also known as WDP, is a microservices based architecture built on the top of [Kubernetes](https://kubernetes.io). WDP is composed by several components which work together to provide a complete data analysis platform.
+Wizzie Data Platform, also known as WDP, is a microservices based architecture built on the top of [Kubernetes](https://kubernetes.io). WDP is composed by several microservices or components which work together to provide a complete data analysis platform.
 
 The WDP architecture and components interrelations are currently defined using a infrastructure as code (IaC) approach. It allows to us to automate the complete infrastructure deployment process in a repeatable and consistent manner, enabling a better testing and quality control, more efficient and predictable deployments, and decreased recovery times.
 
 [Terraform](https://www.terraform.io/) is the chosen solution to reach this goal. The full WDP infrastructure is defined in a [HCL](https://github.com/hashicorp/hcl) project and it is called **Terra-Box**.
```

Comment:
`is defined in a HCL` -> `is defined in an HCL`
`project and it`-> `project, and it`